tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py

```diff
@@ -567,6 +567,44 @@ def test_calculate_nondomination_rank() -> None:
     assert ranks == [0, 1, 0, 2, 0, 1]
 
 
+def test_calculate_weights_below() -> None:
+    sampler = MOTPESampler()
+
+    # Two samples
```

Comment:
[nit] Could you add periods to the end of the code comments? We usually use full sentences as shown in the [contribution guide](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences).